docs/admin/kubeadm-join.md

```diff
@@ -0,0 +1,222 @@
+---
+approvers:
+- mikedanese
+- luxas
+- jbeda
+title: Kubeadm join
+notitle: true
+---
+{% capture overview %}
+## Kubeadm join {#cmd-join}
+{% endcapture %}
+
+{% capture body %}
+{% include_relative _kubeadm/kubeadm_join.md %}
+
+### Join workflow 
+`kubeadm init` bootstraps a Kubernetes master node by executing the
+following steps:
+
+The `kubeadm join` bootstraps a Kubernetes worker node and joins it to the cluster. 
+This action consists of the following steps:
+
+1. kubeadm downloads necessary cluster information from the API server.
+   By default, it uses the bootstrap token and the CA key hash to verify the
+   authenticity of that data. The root CA can also be discovered directly via a
+   file or URL.
+
+1. Once the cluster information is known, kubelet can start the TLS bootstrapping
+   process.
+
+   The TLS bootstrap uses the shared token to temporarily authenticate
+   with the Kubernetes Master to submit a certificate signing request (CSR); by
+   default the control plane will sign this CSR request automatically.
+
+1. Finally, kubeadm will configure the local kubelet to connect to the API
+   server with the definitive identity assigned to the node.
+
+### Discovery
+The kubeadm discovery has several options, each with security tradeoffs.
+The right method for your environment depends on how you provision nodes and the
+security expectations you have about your network and node lifecycles.
+
+#### Token-based discovery with CA pinning
+_This is the default mode in Kubernetes 1.8._ In this mode, kubeadm downloads
+the cluster configuration (including root CA) and validates it using the token
+as well as validating that the root CA public key matches the provided hash and
+that the API server certificate is valid under the root CA.
+
+**Example `kubeadm join` command:**
+
+ - `kubeadm join --discovery-token abcdef.1234567890abcdef --discovery-token-ca-cert-hash sha256:1234..cdef 1.2.3.4:6443`
+
```

Comment:
I think this is a command followed by output. If so, separate the output from the command.